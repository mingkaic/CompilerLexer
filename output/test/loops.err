decafast(25674,0x7fff76133000) malloc: *** error for object 0x7fcbb1c03ab0: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
