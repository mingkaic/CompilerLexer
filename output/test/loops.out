Program(None,Package(C,FieldDeclAST(x,IntType,Scalar),FieldDeclAST(size,IntType,Scalar),Method(f,VoidType,None,MethodBlock(VarDef(x,IntType),VarDef(i,IntType),AssignVar(size,NumberExpr(100)),WhileStmtAST(BoolExpr(true),Block(None,AssignVar(x,NumberExpr(1)),AssignVar(leftPtr,NumberExpr(10)),ForStmtAST(AssignVar(i,NumberExpr(0)),BinaryExpr(Lt,VariableExpr(i),VariableExpr(size)),AssignVar(i,BinaryExpr(Plus,VariableExpr(i),NumberExpr(1)))),AssignVar(i,BinaryExpr(Plus,VariableExpr(i),NumberExpr(1)))))))))
